offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/projects/ProjectDbConnector.groovy

```diff
@@ -0,0 +1,164 @@
+package life.qbic.portal.offermanager.dataresources.projects
+
+import groovy.util.logging.Log4j2
+
+import life.qbic.datamodel.dtos.general.Person
+import life.qbic.portal.offermanager.dataresources.persons.PersonDbConnector
+
+import life.qbic.datamodel.dtos.business.*
+import life.qbic.datamodel.dtos.projectmanagement.*
+import life.qbic.business.projects.create.ProjectExistsException
+
+import life.qbic.business.exceptions.DatabaseQueryException
+import life.qbic.portal.offermanager.dataresources.database.ConnectionProvider
+
+import java.sql.Connection
+import java.sql.PreparedStatement
+import java.sql.ResultSet
+import java.sql.Statement
+
+/**
+ * Provides operations on QBiC project data
+ *
+ * This class is responsible for transferring project data to the customer db
+ *
+ * @since: 1.0.0
+ *
+ */
+@Log4j2
+class ProjectDbConnector {
+
+  /**
+   * A connection to the customer database used to create queries.
+   */
+  private final ConnectionProvider connectionProvider
+  private final PersonDbConnector personDbConnector
```

Comment:
the comment should be for the second variable, right?

Reply:
yes, fixed